src/components/article-card.jsx

```diff
@@ -0,0 +1,40 @@
+import {useState, useEffect} from 'react';
+import '../test.css';
+import {getArticleByArticleId} from '../utils/api'
+const ArticleCard = (article) => {
+    const postedAt = article.created_at.slice(0, article.created_at.length -5).replace('T', '  ')
+    const topic = `${article.topic.slice(0,1).toUpperCase()}${article.topic.slice(1, article.topic.length)}`
+    const [articleBody, setArticleBody] = useState('');
+    useEffect(() => {
+        if (articleBody === '') {
+            getArticleByArticleId(article.article_id).then(({body}) => {
+                setArticleBody(`${body.slice(0, 120)}...`)
```

Comment:
This is extra to the ticket which is supposed to focus on getting/rendering a list of all reviews